docs/stargazer.ipynb

```diff
@@ -23,6 +23,7 @@
     "    -   [Display p-values or confidence intervals](#display-p-values-or-confidence-intervals)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    stargazer_table = Stargazer([fit1, fit2, fit3, fit4, fit5, fit6])
</pre><p>This seems to have failed, likely because you're running the "original" stargazer and not the updated clone in <code>py-econometrics</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/py-econometrics/pyfixest/pull/591/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>